shell/platform/darwin/ios/framework/Headers/FlutterPlugin.h

```diff
@@ -110,6 +110,16 @@ NS_ASSUME_NONNULL_BEGIN
             openURL:(NSURL*)url
   sourceApplication:(NSString*)sourceApplication
          annotation:(id)annotation;
+
+/**
+ Called if this plugin has been registered for `UIApplicationDelegate` callbacks.
+
+  - Returns: `YES` if this plugin handles the request.
+*/
+- (BOOL)application:(UIApplication *)application
```

Comment:
The changed files may now need of a trip through the formatter. From `engine/src` you can do
```
./buildtools/toolchain/clang+llvm-x86_64-darwin/bin/clang-format -i flutter/shell/platform/darwin/ios/framework/Headers/FlutterPlugin.h
./buildtools/toolchain/clang+llvm-x86_64-darwin/bin/clang-format -i flutter/shell/platform/darwin/ios/framework/Source/FlutterAppDelegate.mm
```
to overwrite the changed files with formatted versions.